### PR TITLE
Feature/hocs 2611 k8s refinement

### PR DESCRIPTION
**Pull request by philbarton**
Remove unneeded config from deployment.yaml
  Only run one pod
  Delete existing pod before starting new, Using recreate strategy

### CODE REVIEW

File: kd/deployment.yaml [hunk 1]
```diff
@@ -6,15 +6,12 @@ metadata:
   labels:
     version: {{.VERSION}}
 spec:
-  replicas: {{.MIN_REPLICAS}}
+  replicas: 1
```
Comment (zuzak) [inline kd/deployment.yaml:9]: Will we still want `1` replica even in production?

**Review by zuzak — APPROVED** [review]

File: kd/deployment.yaml [hunk 2]
```diff
@@ -172,16 +111,14 @@ spec:
...
-              value: '8080'
-            - name: ENDPOINTS_INFO_ENABLED
-              value: 'false'
             - name: SPRING_PROFILES_ACTIVE
               value: 'sqs, sns, s3'
             - name: CASE_CREATOR_CASE_SERVICE
               value: 'https://hocs-casework.{{.KUBE_NAMESPACE}}.svc.cluster.local'
             - name: CASE_CREATOR_WORKFLOW_SERVICE
               value: 'https://hocs-workflow.{{.KUBE_NAMESPACE}}.svc.cluster.local'
+            - name: CASE_CREATOR_SHUTDOWN_DELAY_SECONDS
```
Comment (zuzak) [inline kd/deployment.yaml:120]: Do you need this env-var anymore?

Comment (philbarton) [reply kd/deployment.yaml:120]: yeh it still sleeps, because without it spring dies on SIGTERM

File: kd/deployment.yaml (hunk 1 above)
Comment (philbarton) [reply kd/deployment.yaml:9]: yeh, with it being a consumer it will get out of hand if there are multiple consumers also throttling will get complex.

**Pull request merged**